{
  "uid" : "c63cc3c9014cb391",
  "name" : "Ensuring log storage and integrity",
  "fullName" : "Logging Capabilities: Ensuring log storage and integrity",
  "historyId" : "e62c34eb79898007cd294c7beb438198",
  "time" : {
    "start" : 1746710238057,
    "stop" : 1746710238626,
    "duration" : 569
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: STEP: When the system restarts\n",
  "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_007_steps.py\", line 109, in step_when_system_restarts\n    raise AssertionError(msg)\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: STEP: When the system restarts\n",
    "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_007_steps.py\", line 109, in step_when_system_restarts\n    raise AssertionError(msg)\n",
    "steps" : [ {
      "name" : "Given the admin user is logged into TPC_Web",
      "time" : {
        "start" : 1746710238059,
        "stop" : 1746710238598,
        "duration" : 539
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "When the system restarts",
      "time" : {
        "start" : 1746710238598,
        "stop" : 1746710238599,
        "duration" : 1
      },
      "status" : "failed",
      "statusMessage" : "AssertionError: STEP: When the system restarts\n",
      "statusTrace" : "  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/model.py\", line 1909, in run\n    match.run(runner.context)\n  File \"/home/runner/work/trustpoint/trustpoint/.venv/lib/python3.12/site-packages/behave/matchers.py\", line 104, in run\n    self.func(context, *args, **kwargs)\n  File \"trustpoint/features/steps/r_007_steps.py\", line 109, in step_when_system_restarts\n    raise AssertionError(msg)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    }, {
      "name" : "Then previous logs are still accessible",
      "time" : {
        "start" : 1746710238626,
        "stop" : 1746710238626,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "And unauthorized users cannot delete or modify logs",
      "time" : {
        "start" : 1746710238626,
        "stop" : 1746710238626,
        "duration" : 0
      },
      "status" : "skipped",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 4,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "severity",
    "value" : "normal"
  }, {
    "name" : "feature",
    "value" : "Logging Capabilities"
  }, {
    "name" : "framework",
    "value" : "behave"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 2,
        "broken" : 0,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 2
      },
      "items" : [ {
        "uid" : "764c4011c89fbece",
        "reportUrl" : "https://Trustpoint-Project.github.io/trustpoint/20/#testresult/764c4011c89fbece",
        "status" : "failed",
        "statusDetails" : "AssertionError: STEP: When the system restarts\n",
        "time" : {
          "start" : 1746709784188,
          "stop" : 1746709784755,
          "duration" : 567
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "c63cc3c9014cb391.json",
  "parameterValues" : [ ]
}